### PR TITLE
S1T4 [Defect_1]: Settings by user id doesn't work

### PR DESCRIPTION
**Pull request by greySlon**

### CODE REVIEW

File: cityelf/src/main/java/com/cityelf/repository/NotificationRepository.java [hunk 1]
```diff
@@ -2,12 +2,22 @@
...
-    return new Notification();
+  @Autowired
+  private UserRepository userRepository;
+
+  public Notification getNotification(long id) {
+    User innerUser = userRepository.getUser(id);
+    if (innerUser != null) {
+      return innerUser.getNotification();
+    } else {
+      return null;
```
Comment (parkhomenko) [inline cityelf/src/main/java/com/cityelf/repository/NotificationRepository.java]: if you return **null** from the method, you should consider Java 8 Optional class

**Review by parkhomenko — CHANGES_REQUESTED** [review]
Correct one point I mentioned and change the commit message. It should be defined for a new defect, not the US you have already submitted before.

**Review by parkhomenko — CHANGES_REQUESTED** [review]
Could you change the commit message to "S1T4 [Defect_1]: Settings by user id doesn't work" as it is written in the card in trello.

**Comment by greySlon** [comment]
готово

**Review by parkhomenko — APPROVED** [review]

**Pull request merged**